client/src/containers/Authenticate/index.js

```diff
@@ -53,81 +53,90 @@ export default function Authenticate({ children }) {
     }
   }
 
-  const auth2Ref = useRef();
-
-  useEffect(() => {
-    // adapted from https://developers.google.com/identity/sign-in/web/build-button
-    const gapi = window.gapi;
-    gapi.load('auth2', () => {
-      // Retrieve the singleton for the GoogleAuth library and set up the client.
-      auth2Ref.current = gapi.auth2.init({
-        client_id:
-          '514830196757-8464k0qoaqlb4i238t8o6pc6t9hnevv0.apps.googleusercontent.com',
-        cookiepolicy: 'single_host_origin',
-        // Request scopes in addition to 'profile' and 'email'
-        //scope: 'additional_scope'
-      });
-      auth2Ref.current.isSignedIn.listen((isSignedIn) => {
-        if (isSignedIn) {
-          const googleUser = auth2Ref.current.currentUser.get();
-          const googleUserProfile = googleUser.getBasicProfile();
-          const name = googleUserProfile.getName();
-          const email = googleUserProfile.getEmail();
-          const id_token = googleUser.getAuthResponse().id_token;
-
-          const newHeaders = new Headers();
-          newHeaders.append('Authorization', `Token ${id_token}`);
-          newHeaders.append('Content-Type', 'application/json');
-          newHeaders.append('Accept', 'application/json');
-
-          dispatch({ type: 'LOGIN_BEGIN' });
-          // Verify the backend API is working and
-          // accepts the id_token
-          return fetch(`/api/person/${email}`, {
-            headers: newHeaders,
-          }).then((response) => {
-            if (response.ok) {
-              response.json().then(({ id }) => {
-                dispatch({
-                  type: 'LOGIN_SUCCESS',
-                  payload: {
-                    user: {
-                      name,
-                      email,
-                      id_token,
-                      id,
-                    },
-                  },
-                });
-              });
-            } else {
-              response.text().then((error) => {
-                dispatch({ type: 'LOGIN_FAILURE', payload: { error } });
-              });
-            }
+  useEffect(
+    () => {
+      console.log(
```

Comment:
Maybe all these logs can be deleted now?

Reply:
Good point, done.